python/Ganga/GPIDev/Schema/Schema.py

```diff
@@ -406,9 +406,6 @@ def isA(self, _what):
 
         what = stripProxy(_what)
 
-        if isinstance(what, types.InstanceType):
```

Comment:
It was intended to check whether a particular variable pointed to an *instance* or a *class*. Unfortunately it did not do what was expected anyway and the test always failed.